mypy/test/teststubgen.py

```diff
@@ -256,6 +256,15 @@ class StubgenPythonSuite(DataSuite):
     files = ['stubgen.test']
 
     def run_case(self, testcase: DataDrivenTestCase) -> None:
+        old_sys_path = sys.path[:]
+        if not ('' in sys.path or '.' in sys.path):
+            sys.path.insert(0, '')
```

Comment:
Ah yes, `stubgen.main()` has this, but the tests don't run `main()`, they typically call more fine grained functions.